src/decryption-requests/DecryptionSender.sol

```diff
@@ -191,14 +204,30 @@ contract DecryptionSender is
      * @dev See {IDecryptionSender-isInFlight}.
      */
     function isInFlight(uint256 requestID) public view returns (bool) {
-        return requestsInFlight[requestID].callback != address(0);
+        return !requests[requestID].isFulfilled;
```

Comment:
for `requestID`s that don't exist, this will return `true` I think, but that's an edge case

Reply:
Resolved.